### PR TITLE
Test branding

### PR DESCRIPTION
**Pull request by wenfeng-xing**

### CODE REVIEW

**Review by iandilling-adaptive — APPROVED** [review]

File: packages/ui-kit-components/src/components/Branding/LogoBrand/LogoBrand.test.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  },
+}));
+
+describe('LogoBrand component tests', () => {
+  describe('Given a default LogoBrand', () => {
+    beforeEach(() => {
+      render(<LogoBrand />);
+    });
+
+    it('Should be the same as the snapshots', () => {
```
Comment (qiqi0725) [inline packages/ui-kit-components/src/components/Branding/LogoBrand/LogoBrand.test.tsx:25]: very minor: depending on how strict the team wants to be, add `Then` 😄

File: packages/ui-kit-components/src/components/Branding/LogoText/LogoText.test.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    it('Then should get the height 90 in the document text', () => {
+      const element = screen.getByTestId('logo-text');
+
+      expect(element).toHaveTextContent('90');
+    });
+  });
+
+  describe('Given a LogoText with customized size', () => {
+    describe('When pass the width 50 and height 100', () => {
```
Comment (qiqi0725) [inline packages/ui-kit-components/src/components/Branding/LogoText/LogoText.test.tsx:47]: similar to what my comment in #249 
`When pass the width 50 and height 100` isn't necessary as `50`, `100` could be anything, are are just customized size

so you could either:
```
Given LogoText
When render with custom props / When render with default props
Then ...
```

or 
```
Given default LogoText
Then ...

Given LogoText with custom props
Then ...
```

File: packages/ui-kit-components/src/components/Branding/Logo/Logo.test.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}));
+
+describe('Logo component test', () => {
+  describe('Given a default Logo', () => {
+    beforeEach(() => {
+      render(<Logo />);
+    });
+
+    describe('When render the default Logo', () => {
+      it('Then should it render correctly', () => {
```
Comment (qiqi0725) [inline packages/ui-kit-components/src/components/Branding/Logo/Logo.test.tsx:27]: Typo: `Then it should`

File: packages/ui-kit-components/src/components/Branding/LogoBrand/LogoBrand.test.tsx [hunk 4]
```diff
@@ -30,13 +30,13 @@ describe('LogoBrand component tests', () => {
       expect(screen.getByTestId('logo-brand')).toBeInTheDocument();
     });
 
-    it('Then should get the same width as default in the document text', () => {
+    it('Then with should return the right width value', () => {
```
Comment (qiqi0725) [inline packages/ui-kit-components/src/components/Branding/LogoBrand/LogoBrand.test.tsx:33]: typo: with -> width

same for following tests

**Review by qiqi0725 — APPROVED** [review]

**Pull request merged**